frontend/package-lock.json

```diff
@@ -19851,7 +19851,8 @@
         },
         "ssri": {
           "version": "6.0.1",
-          "resolved": "",
+          "resolved": "https://registry.npmjs.org/ssri/-/ssri-6.0.1.tgz",
+          "integrity": "sha512-3Wge10hNcT1Kur4PDFwEieXSCMCJs/7WvSACcrMYrNp+b8kDL1/0wJch5Ni2WrtwEa2IO8OsVfeKIciKCDx/QA==",
```

Comment:
I don't know what this is. Can you tell me?

Reply:
This was automatic, I think when I did `npm install`. This is the reason I wanted to add this file to gitignore so as to avoid noise to our diffs on Github. It's a preference so I have no problem fixing the pr and removing the changes